File: Skip_Backend/src/main/java/com/simplife/skip/payload/requests/ViajeRequest.java
package com.simplife.skip.payload.requests;

import com.simplife.skip.models.Parada;
import lombok.Data;

import java.util.List;

@Data
public class ViajeRequest {

    private Long conductorId;
    private String sentidoRuta;
    private List<Parada> paradas;
    private String tiempoEstimado;
    private String descripcion;
    private String fechaViaje;
    private String horaInicio;
    private String horaLlegada;

}

File: Skip_Android/app/src/main/java/com/simplife/skip/models/ViajeRequest.kt
package com.simplife.skip.models

import java.io.Serializable

class ViajeRequest (
    var conductorId: Long,
    //True - hacia la universidad
    //False - desde la universidad
    var sentidoRuta: Boolean,
    var partida: Parada,
    var destino: Parada,
    var tiempoEstimado: String,
    var distancia: Float,
    var descripcion: String,
    var fechaViaje: String,
    var horaInicio: String,
    var horaLlegada: String
) : Serializable {
}